lib/business/notifications.dart

```diff
@@ -63,6 +63,7 @@ class Notifications {
 
   add(EnvoyNotification notification) {
     notifications.add(notification);
+    notifications.sort((a, b) => b.date.compareTo(a.date));
```

Comment:
Probably better to do this at the point of display. 

IMO it's good to separate internal representation of information from the way information is presented to the user.

We also might do more types of (perhaps dynamic) sorting, filtering on the notifications in the future.

Reply:
Understood.